src/index.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+
+interface BaseResponse {
+    message: string;
+    data: any;
+}
+
+export class Enforcer {
+    private endpoint: string;
+    private user : string | undefined;
+    private profiles: {[key: string]: string[]} = {};
```

Comment:
I think it's better to use `Map`.

Reply:
Use `Map` at the frontend is reasonable. Actually, I will implement a class `Profiles` for manipulate the data and relevant methods.